### PR TITLE
Refactor: clean call view styles and use CSS variables.

### LINKED ISSUE #12910: [refactor] Use CSS variables in a Call View
- ref: https://github.com/nextcloud/spreed/pull/12875#discussion_r1705741077

### PR DESCRIPTION
**Pull request by DorraJaouad**
### ☑️ Resolves

* Fix #12910

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image
image | image
image | image

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/CallView/Grid/Grid.vue [hunk 1]
```diff
@@ -1010,19 +1011,14 @@ export default {
 	height: 100%;
 	width: 100%;
 
-	grid-row-gap: 8px;
-	grid-column-gap: 8px;
+	grid-row-gap: var(--grid-gap);
```
Comment (Antreesy) [inline src/components/CallView/Grid/Grid.vue:1014]: I remember there was a plan to add CSS variables for 8/12/16px. Maybe stick with calc value, to migrate easier later?

File: src/components/CallView/shared/VideoVue.vue [hunk 2]
```diff
@@ -48,8 +52,11 @@
 					:source="participantActorType"
 					:size="avatarSize"
 					disable-menu
-					disable-tooltip
-					:class="avatarClass" />
+					disable-tooltip />
+				<NcLoadingIcon v-if="isLoading"
```
Comment (Antreesy) [inline src/components/CallView/shared/VideoVue.vue:56]: For me it makes more sense to integrate it into `AvatarWrapper` component, with `loading` prop, maybe?

File: src/components/CallView/Grid/Grid.vue (hunk 1 above)
Comment (DorraJaouad) [reply src/components/CallView/Grid/Grid.vue:1014]: Well, we can split into 3 local variables and rename it later to CSS variables.

File: src/components/CallView/shared/VideoVue.vue (hunk 2 above)
Comment (DorraJaouad) [reply src/components/CallView/shared/VideoVue.vue:56]: Sounds like a good idea 👍🏻

File: src/components/CallView/Grid/Grid.vue (hunk 1 above)
Comment (Antreesy) [reply src/components/CallView/Grid/Grid.vue:1014]: Whatever you find a good solution in that case

**Review by Antreesy — APPROVED** [review]

**Pull request merged**